app/src/main/java/org/fossasia/openevent/app/events/EventListFragment.java

```diff
@@ -1,162 +1,138 @@
 import android.support.v7.widget.DefaultItemAnimator;
 import android.support.v7.widget.DividerItemDecoration;
 import android.support.v7.widget.LinearLayoutManager;
 import android.support.v7.widget.RecyclerView;
-import android.view.Menu;
-import android.view.MenuItem;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ProgressBar;
 import android.widget.Toast;
 
-import com.android.databinding.library.baseAdapters.BR;
 import com.timehop.stickyheadersrecyclerview.StickyRecyclerHeadersDecoration;
 
 import org.fossasia.openevent.app.OrgaApplication;
 import org.fossasia.openevent.app.R;
+import org.fossasia.openevent.app.common.BaseFragment;
 import org.fossasia.openevent.app.data.contract.IUtilModel;
 import org.fossasia.openevent.app.data.models.Event;
-import org.fossasia.openevent.app.databinding.ActivityEventListBinding;
-import org.fossasia.openevent.app.event.detail.EventDetailActivity;
 import org.fossasia.openevent.app.events.contract.IEventsPresenter;
 import org.fossasia.openevent.app.events.contract.IEventsView;
-import org.fossasia.openevent.app.login.LoginActivity;
-import org.fossasia.openevent.app.utils.Utils;
+import org.fossasia.openevent.app.main.listeners.OnEventLoadedListener;
 import org.fossasia.openevent.app.utils.ViewUtils;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import javax.inject.Inject;
 
+import butterknife.BindView;
+import butterknife.ButterKnife;
 import timber.log.Timber;
 
 /**
- * An activity representing a list of Events. This activity
- * has different presentations for handset and tablet-size devices. On
- * handsets, the activity presents a list of items, which when touched,
- * lead to a {@link EventDetailActivity} representing
- * item details. On tablets, the activity presents the list of items and
- * item details side-by-side using two vertical panes.
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * Use the {@link EventListFragment#newInstance} factory method to
+ * create an instance of this fragment.
  */
-public class EventListActivity extends AppCompatActivity implements IEventsView {
-
-    /**
-     * Whether or not the activity is in two-pane mode, i.e. running on a tablet
-     * device.
-     */
-    private boolean isTwoPane;
-
-    private final List<Event> events = new ArrayList<>();
-    private EventsListAdapter eventListAdapter;
-    private RecyclerView.AdapterDataObserver adapterDataObserver;
-
-    public static final String EVENT_KEY = "event";
-    public static final String EVENT_NAME = "event_name";
+public class EventListFragment extends BaseFragment implements IEventsView{
+    @BindView(R.id.swipeContainer)
+    SwipeRefreshLayout refreshLayout;
+    @BindView(R.id.eventRecyclerView)
+    RecyclerView recyclerView;
+    @BindView(R.id.progressBar)
+    ProgressBar progressBar;
+    @BindView(R.id.emptyView)
+    View emptyView;
```

Comment:
The point is to not cause changes which are to be rechanged again in future, plus, databinding is more efficient and we'll have to shift to it in near future for transient field, so why change a layout already in data binding back to a simple one? If it would be already not in data binding, I wouldn't be suggesting it to be made that way. This is like undoing something only to redo later, involving more work both now and then